### PR TITLE
[IMP] sales: update quote_template for 16

### PR DESCRIPTION
**Pull request by StraubCreative**
Task [3498323](https://www.odoo.com/web#id=3498323&cids=3&model=project.task&view_type=form)
Replaces #5368

### CODE REVIEW

**Review by meng-odoo — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! Nice job on this doc. My suggestions are mainly for removing the Recurrence section and linking to other docs, and making some of the images clearer. Otherwise it's mostly small wording and formatting suggestions. Please let me know if you have any questions! Thanks

**Comment by ksc-odoo** [comment]
@meng-odoo thank you SO much for the always-helpful feedback. I made the suggested edits you gave me, and believe it's ready for the next stage...with your "blessing," of course 👍 Thanks again!

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/quote_template.rst:163]: "Compatible" might sound like some products are not eligible to be optional products if they're not compatible
```suggestion
In the :guilabel:`Optional Products` tab, :guilabel:`Add a line` for each cross-selling
```

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/quote_template.rst:172]: This paragraph is a bit redundant with the previous one---shortening it would help with that
```suggestion
Clicking :guilabel:`Add a line` reveals a blank field in the :guilabel:`Product` column.
```

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/quote_template.rst:214]:
```suggestion
contain (e.g. :guilabel:`Template Header`, :guilabel:`Product`).
```

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/quote_template.rst:257]:
```suggestion
see how the quotation template appears on the front-end of the website through Odoo's customer portal.
```

**Review by meng-odoo — APPROVED** [review]
Hi @ksc-odoo, this doc looks great! It's very clear and thorough. I had a few tiny suggestions, but this is definitely ready to move forward, so I'm approving now :)

**Comment by ksc-odoo** [comment]
Thank you for your always-helpful feedback @meng-odoo -- I made all the suggested edits, so I'll now be tagging the Materials team for a 2nd Peer Review 👍 @hojo-odoo @Felicious @brse-odoo

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (brse-odoo) [inline content/applications/sales/sales/send_quotations/quote_template.rst:144]:
```suggestion
Then, drag-and-drop the note to the desired position, via the :guilabel:`six squares` icon.
```
I feel like we've been told where it is a few times now, so to avoid redundancy and keep it short/sweet, I deleted it here. But, it's just a suggestion so ignore if you'd like to keep it

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (brse-odoo) [inline content/applications/sales/sales/send_quotations/quote_template.rst:219]:
```suggestion
To edit the content, appearance, and overall design of the quotation template via the *Website*
```

**Review by brse-odoo — COMMENTED** [review]
Hi @ksc-odoo, I've finished my peer content review of this doc. Great job! This was really tight and polished, so I only had a few suggestions - mostly grammatical, for you to accept/reject as you see fit.

Feel free to ping me with any questions about my comments!

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/quote_template.rst:15]: I totally understand your suggestion here, but I think it's actually alright in its current state. It's intended to be read as "Go to X,Y,Z -- and scroll to A,B,C" -- does that make sense. However, if you think it's really in need of a change, just let me know, and I'll go back and make the necessary change. Thanks!

Comment (brse-odoo) [reply content/applications/sales/sales/send_quotations/quote_template.rst:15]: This was more so a preference thing as I was reading it, but it works just fine the way you have it so you're good to keep it that way!

**Comment by ksc-odoo** [comment]
Thanks for the review @brse-odoo -- once I get your approval, I'll pass this PR along to final review. Thanks again! 👍

**Comment by brse-odoo** [comment]
> Thanks for the review @brse-odoo -- once I get your approval, I'll pass this PR along to final review. Thanks again! 👍

@ksc-odoo The changes look great! I say this one is ready to send to final review 👍

**Comment by ksc-odoo** [comment]
Thanks @brse-odoo -- Alrighty, @odoo/us-doc-review -- this one's ready for Final Review! 👍

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/quote_template.rst:168]:
```suggestion
ideally complement the original offering as added value for the prospective buyer.
```

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/quote_template.rst:189]: Just a bit redundant from the previous sentence
```suggestion
quotation template. To add terms and conditions, simply type (or
```

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/quote_template.rst:213]: We may want to state explicitly somewhere in this section (maybe as a note?) that this feature is only available if you have the Website app activated

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/quote_template.rst:252]:
```suggestion
the :guilabel:`Quotation Template` field.
```

**Review by meng-odoo — APPROVED** [review]
Great job on this doc @ksc-odoo! I gave it a final review and had very few suggestions. Please move this to technical review once you've gone through it. Thank you!

**Comment by ksc-odoo** [comment]
Thanks for the awesome suggestions, @meng-odoo -- I really appreciate it. I made all those necessary adjustments, and believe this one's ready for Technical Review @samueljlieber 👍

**Comment by StraubCreative** [comment]
@robodoo delegate=samueljlieber

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/quote_template.rst:183]: Optional, but I think being specific here is helpful 🙂
```suggestion
   Optional products are **not** required to create a quotation template.
```

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/quote_template.rst:197]: Can remove single bullet point from seealso, and update doc path to be relative
```suggestion
   :doc:`../../../finance/accounting/customer_invoices/terms_conditions`
```

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/quote_template.rst:200]: Optional, but I think being specific here is helpful 🙂
```suggestion
   Terms and conditions are **not** required to create a quotation template.
```

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/quote_template.rst:242]: Update doc path to be relative
```suggestion
   :doc:`../../../websites/website` documentation to learn more.
```

File: content/applications/sales/sales/send_quotations/quote_template.rst
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/quote_template.rst:270]: Update doc paths to be relative
```suggestion
   - :doc:`get_signature_to_validate`
   - :doc:`get_paid_to_validate`
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! Great job improving the content of this doc, I think you covered designing quote templates very well. I have a few technical corrections and suggestions for you, please see below. Once these changes are addressed tag me once more and I can move this PR forward! Thank you 🙏

**Comment by ksc-odoo** [comment]
Thanks for the super-helpful feedback @samueljlieber -- I made all the suggested edits, and believe this one's ready for another look from you whenever you get a chance 👍

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**
